apps/deploy-web/src/components/authorizations/AllowanceGrantedRow.tsx

```diff
@@ -4,29 +4,33 @@ import { FormattedTime } from "react-intl";
 
 import { Address } from "@src/components/shared/Address";
 import { AKTAmount } from "@src/components/shared/AKTAmount";
+import { Checkbox } from "@src/components/ui/checkbox";
 import { TableCell, TableRow } from "@src/components/ui/table";
 import { AllowanceType } from "@src/types/grant";
 import { getAllowanceTitleByType } from "@src/utils/grants";
 import { coinToUDenom } from "@src/utils/priceUtils";
 
 type Props = {
-  allowance: AllowanceType;
+  allowance: AllowanceType | { granter: "Connected Wallet" };
   children?: ReactNode;
+  onSelect?: () => void;
+  selected?: boolean;
 };
 
-export const AllowanceGrantedRow: React.FunctionComponent<Props> = ({ allowance }) => {
+export const AllowanceGrantedRow: React.FunctionComponent<Props> = ({ allowance, selected, onSelect }) => {
+  const isFull = "allowance" in allowance;
```

Comment:
How does that work? If there's an allowance, it's full?

Reply:
after u asked this question decided to do it a bit differently. Hopefully simpler



https://github.com/akash-network/cloudmos/pull/238/files#diff-3a051be7611032924e7ddd61b9c21e5b15bfdfd97fef2f5e76f614c825ef6758R275

Reply:
Ah clearer this way! 👍